lib/output.ts

```diff
@@ -1,24 +1,32 @@
 import chalk from 'chalk';
 import type { MismatchingDependencyVersions } from './dependency-versions.js';
 import { compareRanges } from './dependency-versions.js';
+import { table } from 'table';
 
-export function mismatchingVersionsToOutputLines(
+export function mismatchingVersionsToOutput(
   mismatchingDependencyVersions: MismatchingDependencyVersions
-): string[] {
-  return mismatchingDependencyVersions.map(
-    (obj) =>
-      `${chalk.bold(obj.dependency)} has more than one version:\n${obj.versions
+): string {
+  if (mismatchingDependencyVersions.length === 0) {
+    throw new Error('No mismatching versions to output.');
```

Comment:
Should an error be thrown when there are no mismatches?

Reply:
Yes because `mismatchingVersionsToOutput` should only be called when there are mismatching versions (that's how the code is written now). When there are no mismatches, there is no output.